docs/pages/index.page.tsx

```diff
@@ -248,14 +248,14 @@ const Page: NextPage<PageProps> = ({ currentVersion, documentTree }) => {
                           href={href}
                           target="_blank"
                           flexBasis={{
-                            base: "calc((100% - var(--ui-space) * 3) / 4)",
-                            md: "calc((100% - var(--ui-space) * 2) / 3)",
-                            sm: "calc((100% - var(--ui-space) * 1) / 2)",
+                            base: "calc((100% - $space * 3) / 4)",
+                            md: "calc((100% - $space * 2) / 3)",
+                            sm: "calc((100% - $space * 1) / 2)",
                           }}
                           maxW={{
-                            base: "calc((100% - var(--ui-space) * 3) / 4)",
-                            md: "calc((100% - var(--ui-space) * 2) / 3)",
-                            sm: "calc((100% - var(--ui-space) * 1) / 2)",
+                            base: "calc((100% - $space * 3) / 4)",
+                            md: "calc((100% - $space * 2) / 3)",
+                            sm: "calc((100% - $space * 1) / 2)",
                           }}
                           bg="white"
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, it should be commented out.

Because there are no supporters.